src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,79 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    @OneToOne
+    @JoinColumn(name = "user_id")
+    @MapsId
```

Comment:
User can place a lot of orders
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "user_id")
```